### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by shu-mutou**
Update Japanese translation file for #4625

/cc @feloy @seokho-son @hwdef @tanjunchen

### CODE REVIEW

File: i18n/ja/README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Kubernetes Dashboard の和訳を実施するにあたって、下記の配慮
 * 英単語と日本語の間には半角スペースを入れます。
   + (例) ○ `これは English word です。`
   + (例) × `これはEnglish wordです。`
+  + ただし、文頭、句読点の直前直後、では、半角スペースを挿入しません。
+  + (例) × `これは English、Chinese、French です。`
```
Comment (atoato88) [inline i18n/ja/README.md]: Is it bad pattern?
I think it is OK pattern, because no spaces exists after `、`.

Comment (shu-mutou) [reply i18n/ja/README.md]: nice catch!

File: i18n/ja/README.md [hunk 2]
```diff
@@ -51,6 +52,9 @@ Kubernetes Dashboard の和訳を実施するにあたって、下記の配慮
 * 英単語と日本語の間には半角スペースを入れます。
   + (例) ○ `これは English word です。`
   + (例) × `これはEnglish wordです。`
+  + ただし、文頭、句読点の直前直後、では、半角スペースを挿入しません。
+  + (例) ◯ `これは English、Chinese、French です。`
```
Comment (atoato88) [inline i18n/ja/README.md]: `◯` is different from `○`. `○` is used at other place In this doc.
`◯`  : L56
`○` : L50, L53, L64

Comment (shu-mutou) [reply i18n/ja/README.md]: thx. done.

**Comment by atoato88** [comment]
Thank you to quick fix.
/lgtm
/approve

**Pull request merged**